### PR TITLE
Fix linechart, add support for data unit in polar area charts

### PR DESCRIPTION
**Pull request by Tobsucht**

### CODE REVIEW

File: src/main/resources/default/templates/wondergem/page-script.html.pasta [hunk 1]
```diff
@@ -372,7 +372,21 @@
             });
     }
 
-    function polarAreaChart(selector, labels, data) {
+    function polarAreaChart(selector, labels, data, dataUnit) {
+        var options = {
+            startAngle: -0.55 * Math.PI
+        };
+
+        if (!!dataUnit) {
```
Comment (andyHa) [inline src/main/resources/default/templates/wondergem/page-script.html.pasta]: !! ?

File: src/main/resources/default/templates/wondergem/page-script.html.pasta [hunk 2]
```diff
@@ -382,9 +396,7 @@
                 labels: labels
             },
             type: 'polarArea',
-            options: {
-                startAngle: -0.55 * Math.PI
```
Comment (andyHa) [inline src/main/resources/default/templates/wondergem/page-script.html.pasta]: why is that dropped?

Comment (Tobsucht) [reply src/main/resources/default/templates/wondergem/page-script.html.pasta]: just moved to the top of the function

File: src/main/resources/default/templates/wondergem/page-script.html.pasta (hunk 1 above)
Comment (sabieber) [reply src/main/resources/default/templates/wondergem/page-script.html.pasta]: das ist ein JS Cast zu boolean :)

**Pull request merged**